app/controllers/medicaid/intro_location_controller.rb

```diff
@@ -7,13 +7,5 @@ class IntroLocationController < StandardStepsController
     def ensure_application_present
       true
     end
-
-    def existing_attributes
-      {}
-    end
-
-    def step_class
-      Medicaid::IntroLocation
```

Comment:
the page loads properly without this now?

Reply:
It does.